pkgs/applications/window-managers/gamescope/default.nix

```
@@ -18,15 +19,25 @@
 , seatd
 , xwayland
 , glslang
+, hwdata
+, openvr
 , stb
-, wlroots
+, wlroots_0_16
```

```suggestion
, wlroots_0_16
```
Pinned version are usually inherited from top-level.

---

done